satpy/modifiers/geometry.py

```diff
@@ -173,3 +173,8 @@ def __init__(self, correction_limit=88., **kwargs):
     def _apply_correction(self, proj, coszen):
         logger.debug("Apply the effective solar atmospheric path length correction method by Li and Shibata")
         return atmospheric_path_length_correction(proj, coszen, limit=self.correction_limit, max_sza=self.max_sza)
+
+
+def parallax_correction(sc):
+    """Apply parallax correction to entire scene."""
+    return sc
```

Comment:
This doesn't seem to be doing anything atm, should the docstring be changed accordingly, and maybe a warning be issued?

Reply:
Oops, this was a leftover from an earlier dead-end attempt.  Will remove it for this PR.